src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -72,11 +72,11 @@ abstract class CompilerFleetMessageReceiver : MessageReceiver {
 private val LOGGER = LoggerFactory.getLogger("TaskReceiver")
 
 class TaskReceiver(tasksDirectory: String,
-                   private val resultPublisher: Publisher,
-                   private val callback: (message: String, filename: String) -> Unit
+                   private val resultPublisher: Publisher
 ) : CompilerFleetMessageReceiver() {
-    private val dockerProcessor = DockerProcessor(getDefaultDockerClient())
     private val tasksDir: Path
+    private val dockerProcessor = DockerProcessor(getDefaultDockerClient())
+    private val MOCK_PDF_FILE = File(TaskReceiver::class.java.classLoader.getResource("example.pdf").file)
```

Comment:
You gonna be surprised.  These two lines in `init` method below
```
        println(MOCK_PDF_FILE)
        println("Exist? ${MOCK_PDF_FILE.exists()}")
```

would print the following:

```
[bard@bardtop3 papeeria-compiler-fleet (tkt-26-receive-tasks *)]$ build/install/papeeria-compiler-fleet/bin/papeeria-compiler-fleet --sub foo -r bar --tasks-dir /tmp
file:/home/bard/papeeria-compiler-fleet/build/install/papeeria-compiler-fleet/lib/papeeria-compiler-fleet-1.0-SNAPSHOT.jar!/example.pdf
Exist? false
```

Reply:
Changed file into InputStream